python/Hash Table/hash_table.py

```diff
@@ -0,0 +1,73 @@
+import sys
+sys.path.append('/Data_Structures/')
+from linked_lists.linked_list import LinkedList
```

Comment:
Cannot find the module named **'linked_lists'**

Reply:
about the module "Linked_lists" , I have deleted that file jn third commit. Sorey I didn't mentioned in the commit message.